apps/examples/lcd_test/example_lcd.c

```diff
@@ -254,13 +254,94 @@ static void test_bit_map(void)
 	free(lcd_data);
 }
 
+static int test_fps_helper(int color){
+	int fd = 0, fd_lcd = 0, p = 0;
+	char port[20] = { '\0' };
+	size_t len;
+
+	fd = open("/dev/rtc0", O_RDWR);
+	if (fd < 0) {
+		printf("ERROR: LCD FPS test, Fail to open rtc.\n");
+		return;
+	}
+
+	len = xres * yres * 2 + 1;
+	uint8_t *lcd_data = (uint8_t *)malloc(len);
+	if (lcd_data == NULL) {
+		printf("malloc failed for lcd data : %d\n", len);
+		return;
+	}
+	for (int i = 0; i < xres * yres * 2; i += 2) {
+		lcd_data[i] = (color & 0xFF00) >> 8;
+		lcd_data[i + 1] = color & 0x00FF;
+	}
+
+	struct lcddev_area_s area;
+	area.planeno = 0;
+	area.row_start = 0;
+	area.row_end = yres - 1;
+	area.col_start = 0;
+	area.col_end = xres - 1;
+	area.stride = 2 * xres;
+	area.data = lcd_data;
+
+	sprintf(port, LCD_DEV_PATH, p);
+	fd_lcd = open(port, O_RDWR | O_SYNC, 0666);
+	if (fd_lcd < 0) {
+		printf("ERROR: Failed to open lcd port : %s error:%d\n", port, fd_lcd);
+		free(lcd_data);
+		return;
+	}
+
+	struct rtc_time start_time = RTC_TIME_INITIALIZER(1970, 1, 1, 0, 0, 0);
+	struct rtc_time end_time = RTC_TIME_INITIALIZER(1970, 1, 1, 0, 0, 0);
+
+	ioctl(fd, RTC_SET_TIME, (unsigned long)&start_time);
+	ioctl(fd, RTC_SET_TIME, (unsigned long)&end_time);
+
+	ioctl(fd, RTC_RD_TIME, (unsigned long)&start_time);
+	for(int itr = 0; itr < FPS_TEST_FRAME_COUNT; itr++){
```

Comment:
1. coding rule
```
for (...) {
```
2. I don't know how lcd works with non-changed data (color). Are you able to change the color at every loop? For exampe, print red -> print yellow -> print red like that.

Reply:
I am switching between red and blue for 1000 loop.